quarkus/tests/junit5/src/main/java/org/keycloak/it/utils/Maven.java

```diff
@@ -57,16 +57,12 @@ public static Path resolveArtifact(String groupId, String artifactId) {
             ArtifactDescriptorResult projectDescriptor = repositorySystem.readArtifactDescriptor(
                     ctx.getRepositorySystemSession(),
                     new ArtifactDescriptorRequest()
-                            .setArtifact(new DefaultArtifact(project.getGroupId(), project.getArtifactId(),                                    "pom", project.getVersion()))
+                            .setArtifact(new DefaultArtifact(project.getGroupId(), project.getArtifactId(), "pom", project.getVersion()))
                             .setRepositories(remoteRepositories));
             List<Dependency> dependencies = new ArrayList<>(projectDescriptor.getDependencies());
             dependencies.addAll(projectDescriptor.getManagedDependencies());
             Artifact artifact = resolveArtifact(groupId, artifactId, dependencies);
 
-            if (artifact == null) {
```

Comment:
@vaceksimon Are you sure this should be deleted?

Reply:
I think it should, along with the line below. Since `projectDescriptor.getManagedDependencies()` is added to the `dependencies` list on line 63 and checked already on line 64.

```
            dependencies.addAll(projectDescriptor.getManagedDependencies());
            Artifact artifact = resolveArtifact(groupId, artifactId, dependencies);
```